internal/cluster/cluster.go

```diff
@@ -1399,8 +1399,12 @@ func (m *Manager) setConnectivityMajorityGroupsForClusterInternal(cluster *commo
 		majorityGroups[cidr] = majorityGroup
 	}
 
+	primaryMachineCIDR := ""
```

Comment:
This will result in an error (from `IpInCidr()`) if the Machine CIDRs are not defined yet. But I presume what we actually want is for it to fall back to the previous behaviour in those cases.

Reply:
Good point, maybe if machine cidrs are not defined, then this validation shouldn't pass?

Reply:
No, I think changing it to fall back to the old behaviour (i.e. not filter IPs if there are no machine networks) was the right thing to do, for the reasons given in https://github.com/openshift/assisted-service/pull/6094#pullrequestreview-1950748577

Reply:
Great, this change should be in place then. Thank you for your input!